src/modal/modal.component.ts

```diff
@@ -148,6 +148,12 @@ export class ModalComponent implements OnInit, OnDestroy {
 	 */
 	modalState = "out";
 
+	/**
+	 * An element should have 'data-modal-primary-focus' as an attribute to receive initial focus within the `Modal` component.
+	 * @memberof ModalComponent
+	 */
+	selectorPrimaryFocus = "[data-modal-primary-focus]";
```

Comment:
since this all passes through the angular template parser we should be okay with just `[modal-primary-focus]` ... thoughts?

Reply:
Weighing in - I guess it's an attempt to align to our vanilla code, though such alignment is not a hard requirement.

Reply:
Yes I see this selector is used in both vanilla and react code so thought it best to use the same here to make it easier for people transitioning between the different implementations.

Reply:
We don't use the `data-` prefix anywhere else unless required by the carbon css.